src/validation/rules/SingleFieldSubscriptionsRule.ts

```diff
@@ -9,11 +9,14 @@ import type {
 import { Kind } from '../../language/kinds';
 
 import type { ValidationContext } from '../ValidationContext';
+
 import type { ExecutionContext } from '../../execution/execute';
+
```

Comment:
Thanks for fixing those 👍 
I extracted this change into #3167
Can you please rebase?

Reply:
Will do

Reply:
Done